tests/groupTree.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import sunbeam
+import datetimen
+
+spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+
+
+class TestGroupTree(unittest.TestCase):
+    spe3 = None
+
+    def setUp(self):
+        if self.spe3 is None:
+            self.spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+        time = es.schedule.start
+        self.groupTree = self.spe3.schedule.groupTree(time)
+
+    def test(self):
+        group = self.schedule.group('G1')
+        parent = self.groupTree.parent(group)
+        children = self.groupTree.children(group)
+
+        self.assertEqual('G0', parent)
```

Comment:
I might misunderstand here - but when I look at the SPE3 case it looks to me like the group tree should be:
```
            FIELD
              |
              G1
        ----------------
        |              |
       PROD           INJ 
```
i.e. I do not really understand where the group names 'G0' and 'G2' come from?